op-e2e/sequencer_failover_setup.go

```diff
@@ -299,26 +299,46 @@ func sequencerCfg(rpcPort int) *rollupNode.Config {
 	}
```

Comment:
As a followup I'd suggest moving most if not all of these setup functions into a helper struct in its own package like we have for the dispute game (https://github.com/ethereum-optimism/optimism/blob/e9172f60bbce209c058b916b689ef9ab150647e1/op-e2e/e2eutils/disputegame/helper.go#L73).  The helper struct can hold onto a bunch of useful things like the system to retrieve clients from which makes the actual test code much simpler.  Being in a separate package lets you use visibility rules to hide implementation details from the tests (and avoid a lot of name conflicts).